### PR TITLE
[wrt] Update manifest and suite json for app tools support

### PR DESCRIPTION
**Pull request by kangxu**
Impacted TCs num(approved): New 0, Update 134, Delete 1
Unit test Platform: Crosswalk Project for Android 17.45.430.0
Unit test result summary: Pass 131, Fail 3, Blocked 0

https://crosswalk-project.org/jira/browse/XWALK-5760
https://crosswalk-project.org/jira/browse/XWALK-5714
https://crosswalk-project.org/jira/browse/XWALK-5716
https://crosswalk-project.org/jira/browse/XWALK-5799
https://crosswalk-project.org/jira/browse/XWALK-5726
https://crosswalk-project.org/jira/browse/XWALK-5724

### CODE REVIEW

**Comment by kangxu** [comment]
Continue to update for other suites

File: wrt/wrt-internetstdmanu-android-tests/internetstdmanu/Crosswalk_WebApp_HttpAuth_Failure.html [hunk 1]
```diff
@@ -41,8 +41,8 @@
       <strong>Test steps:</strong>
     </p>
     <ol>
-      <li>Package a web app using the package command, for example:"python make_apk.py --package=com.httpauthentication --name=HttpAuthentication --app-url=http://httpbin.org/basic-auth/user/passwd" </li>
-      <li>Install the apk via command "adb install Httpauthentication.apk"</li>
+      <li>Package a web app using the package command, for example:"crosswalk-pkg -m '{"name": "HttpAuthentication", "xwalk_package_id": "org.xwalk.httpauthentication","start_url": "http://httpbin.org/basic-auth/user/passwd"}'" -t x86 -c /path/to/crosswalk**.zip /path/to/apk_saved_directory</li>
+      <li>Install the apk via command "adb install org.xwalk.httpauthentication*.apk"</li>
```
Comment (Honry) [inline wrt/wrt-internetstdmanu-android-tests/internetstdmanu/Crosswalk_WebApp_HttpAuth_Failure.html]: No need to provide the command of app install. The apk name maybe updated as the last portion of package id, see XWALK-5658.

**Comment by kangxu** [comment]
@Honry I have updated it following your comments

**Comment by Honry** [comment]
LGTM, @haoxli, PTAL

**Comment by haoxli** [comment]
lgtm

**Pull request merged**